src/pages/PlanPage/subRoutes/CurrentOrgPlan/CurrentPlanCard/PaidPlanCard/PaidPlanCard.tsx

```diff
@@ -57,12 +56,8 @@ function PaidPlanCard() {
         <div className="flex flex-col border-t pt-2 sm:border-0 sm:p-0">
           <p className="mb-2 text-xs font-semibold">Pricing</p>
           <div className="mb-4">
-            {value && baseUnitPrice ? (
-              <PlanPricing
-                plan={plan}
-                value={value}
-                baseUnitPrice={baseUnitPrice}
-              />
+            {baseUnitPrice ? (
```

Comment:
technically you don't need base UnitPrice OR plan here for this component to render, but I left this as is to not accidentally break something